src/ui/component/claimList/view.jsx

```diff
@@ -10,6 +11,7 @@ import usePersistedState from 'effects/use-persisted-state';
 
 const SORT_NEW = 'new';
 const SORT_OLD = 'old';
+const PADDING_ALLOWANCE = 100;
```

Comment:
is this value dependent on other elements on the page, or is this just used as a somewhat arbitrary threshold to initiate the scroll bottom behavior?

if dependent on elements on the page, please calculate their size dynamically

Reply:
@kauffj It looks like '.main-wrapper' is 28 top and bottom padding (+56) padding larger than '.main'. I could tighten that up if it's important but I don't trust it to be exactly 56 because some elements cause there to be decimals (64.39 or 64.40 for the header, not sure why that changes).